model/pod/podutil.go

```diff
@@ -0,0 +1,105 @@
+package pod
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/pkg/errors"
+)
+
+// Constants representing default curl retry arguments.
+const (
+	curlDefaultNumRetries = 10
+	curlDefaultMaxSecs    = 100
+)
+
+// CurlCommand returns the command to curl the evergreen client and retries the request with the default retry parameters.
```

Comment:
AgentScript returns the script to provision and run the agent in the pod's container.